examples/viewer.py

```diff
@@ -0,0 +1,152 @@
+from magnum import gl, shaders
```

Comment:
Should this go under `habitat_sim/utils` alongside viz_utils.py?

Reply:
Agreed. Let's put this in the utils.